assets/javascripts/modules/ajaxFormSubmit.js

```diff
@@ -128,12 +130,27 @@ var ajaxFormSubmit = {
   serializeForAjax: function(formScope) {
     var ret = ['isajax=true'];
     $.each($(formScope).find(':input'), function() {
-      ret.push(encodeURIComponent(this.name) + '=' + encodeURIComponent($(this).val()));
+      var input = $(this);
+      if (!input.data('ajax-header')) {
+        ret.push(encodeURIComponent(this.name) + '=' + encodeURIComponent(input.val()));
+      }
     });
 
     return ret.join('&').replace(/%20/g, '+').replace(/=$/, '').replace(/&$/, '');
   },
 
+  headerFromInput: function(formScope) {
+    var ret = {};
```

Comment:
what is `ret` ? bar being the `return` object? is it a `header`? feels like this could be more self documenting